content/en/docs/concepts/containers/images.md

```diff
@@ -367,7 +367,31 @@ you must ensure all nodes in the cluster have the same pre-pulled images.
 This can be used to preload certain images for speed or as an alternative to authenticating to a
 private registry.
 
-All pods will have read access to any pre-pulled images.
+All pods will have read access to any pre-pulled images, unless the 
+[KubeletEnsureSecretPulledImages](#kubeletensuresecretpulledimages) feature gate
+is enabled.
+
+#### Pre-pulled image authentication {#prepulledimageauthentication}
+
+{{< feature-state feature_gate_name="KubeletEnsureSecretPulledImages" >}}
+
+If the `KubeletEnsureSecretPulledImages` 
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+is enabled, Kubernetes will require valid authentication credentials for every pod
+image, even if that image is already present on the node. This is controlled by the
+`ImagePullcredentialsVerificationPolicy` option in the
+[Kubelet config](/docs/reference/config-api/kubelet-config.v1beta1#ImagePullCredentialsVerificationPolicy).
+
+This configuration controls when image pull credentials must be verified if the
+image is already present on the node. The `NeverVerify` setting mimics the behavior
+of having this feature gate disabled, while the `NeverVerifyPreloadedImages` setting
```

Comment:
The enumaration of possible values should be a list to make it a bit easier to consume.

There are 4 different forms of the policy, I see 2 being documented here.